app/src/scripts/nav/navbar.jsx

```diff
@@ -162,22 +162,54 @@ class BSNavbar extends Reflux.Component {
     } else {
       return (
         <div className="navbar-right sign-in-nav-btn">
-          <button
-            className="btn-blue half"
-            onClick={userStore.googleSignIn.bind(null)}>
-            <i className="fa fa-google" />
-            <span> Sign in</span>
-          </button>
-          <button
-            className="btn-blue half"
-            onClick={userStore.orcidSignIn.bind(null)}>
-            <span className="icon"><img alt="ORCID" width="16" height="16" src="https://orcid.org/sites/default/files/images/orcid_24x24.png" /></span>
-            <span> Sign in</span>
-          </button>
+          <div
```

Comment:
I think this can be simpler if it reuses the existing DropdownButton component and styles (like the user menu), especially since it's in the same spot as that menu.